src/snowcli/cli/snowpark/function/commands.py

```diff
@@ -293,3 +241,21 @@ def function_drop(
     """Deletes a function from a specified environment."""
     cursor = FunctionManager().drop(identifier=identifier)
     return SingleQueryResult(cursor)
+
+
+def _upload_snowpark_artifact(function_manager, function_identifier, file, overwrite):
```

Comment:
1) Here: `overwrite` is now always `True`, so probably we don't need this arg anymore
2) `FunctionManager # create`: the same about `overwrite` - maybe we don't need this arg anymore. Maybe we should rename it to `FunctionManager # create_or_replace`?

Reply:
You are right, changed